docker/dockerfile.base

```diff
@@ -1,8 +1,8 @@
-FROM ubuntu:22.04
+FROM python:3.12-slim-trixie
```

Comment:
Leaving the rest of the original files alone basically, but this saves us the step of installing python3 through apt-get.